eve/main.yml

```diff
@@ -469,6 +469,32 @@ stages:
             --test-suite "$TESTRAIL_SUITE"
             --test-plan "$TESTRAIL_PLAN"
             "$DESCRIPTION_FILE"
+          haltOnFailure: False
+      - ShellCommand:
+          name: Upload TestRail results
+          env:
+            <<: *env_testrail
+            BASE_URL: "%(prop:artifacts_private_url)s/build_status/"
+            REGEXP: "*.xml"
```

Comment:
more like a "PATTERN" or "WILDCARD", but whatever :innocent:

Reply:
Hmm yes PATTERN I like it, I will change it